.github/workflows/release.yaml

```diff
@@ -28,16 +28,18 @@ jobs:
         with:
           fetch-depth: 0
       - name: Get changed files
-        id: changed-files
+        id: changed-markdown-files
         uses: tj-actions/changed-files@800a2825992141ddde1a8bca8ad394cec34d3188 # tag=v42.0.5
+        with:
+          files: CHANGELOG/**.md
       - name: Get release version
         id: release-version
         run: |
-            if [[ ${{ steps.changed-files.outputs.all_changed_files_count }} != 1 ]]; then
-              echo "1 release notes file should be changed to create a release tag, found ${{ steps.changed-files.outputs.all_changed_files_count }}"
+            if [[ ${{ steps.changed-markdown-files.outputs.all_changed_files_count }} != 1 ]]; then
```

Comment:
Out of curiosity, how did this fix the issue you were seeing if this was already matching the id of the step? Was `changed-files` here being interpreted as something else before?

Reply:
It perhaps was with the logic used when it wasn't the `-markdown` flavor.  I saw this in the examples.  It's possible removing the `| ` and using the same `CHANGELOG/**.md` logic without markdown would work, but this does work and seems like a better fit anyways since indeed we only care about markdown files.